docs/using/specifics.md

```diff
@@ -58,111 +58,15 @@ val rdd = sc.parallelize(0 to 999)
-os.environ['PYSPARK_PYTHON'] = '/usr/bin/python3'
-
-import pyspark
-conf = pyspark.SparkConf()
-
-# point to mesos master or zookeeper entry (e.g., zk://10.10.10.10:2181/mesos)
-conf.setMaster("mesos://10.10.10.10:5050")
-# point to spark binary package in HDFS or on local filesystem on all slave
-# nodes (e.g., file:///opt/spark/spark-2.2.0-bin-hadoop2.7.tgz)
-conf.set("spark.executor.uri", "hdfs://10.10.10.10/spark/spark-2.2.0-bin-hadoop2.7.tgz")
-# set other options as desired
-conf.set("spark.executor.memory", "8g")
-conf.set("spark.core.connection.ack.wait.timeout", "1200")
-
-# create the context
-sc = pyspark.SparkContext(conf=conf)
-
-# do something to prove it works
-rdd = sc.parallelize(range(100000000))
-rdd.sumApprox(3)
-```
-
-#### In a R Notebook
-
-```r
-library(SparkR)
-
-# Point to mesos master or zookeeper entry (e.g., zk://10.10.10.10:2181/mesos)
-# Point to spark binary package in HDFS or on local filesystem on all slave
-# nodes (e.g., file:///opt/spark/spark-2.2.0-bin-hadoop2.7.tgz) in sparkEnvir
-# Set other options in sparkEnvir
-sc <- sparkR.session("mesos://10.10.10.10:5050", sparkEnvir=list(
-    spark.executor.uri="hdfs://10.10.10.10/spark/spark-2.2.0-bin-hadoop2.7.tgz",
-    spark.executor.memory="8g"
-    )
-)
-
-# do something to prove it works
-data(iris)
-df <- as.DataFrame(iris)
-head(filter(df, df$Petal_Width > 0.2))
-```
-
-#### In a Spylon Kernel Scala Notebook
-
-```python
-%%init_spark
-# Configure the location of the mesos master and spark distribution on HDFS
-launcher.master = "mesos://10.10.10.10:5050"
-launcher.conf.spark.executor.uri=hdfs://10.10.10.10/spark/spark-2.2.0-bin-hadoop2.7.tgz
-```
-
-```scala
-// Now run Scala code that uses the initialized SparkContext in sc
-val rdd = sc.parallelize(0 to 999)
-rdd.takeSample(false, 5)
-```
-
-#### In an Apache Toree Scala Notebook
-
-The Apache Toree kernel automatically creates a `SparkContext` when it starts based on configuration information from its command line arguments and environment variables. You can pass information about your Mesos cluster via the `SPARK_OPTS` environment variable when you spawn a container.
-
-For instance, to pass information about a Mesos master, Spark binary location in HDFS, and an executor options, you could start the container like so:
-
-```
-docker run -d -p 8888:8888 -e SPARK_OPTS='--master=mesos://10.10.10.10:5050 \
-    --spark.executor.uri=hdfs://10.10.10.10/spark/spark-2.2.0-bin-hadoop2.7.tgz \
-    --spark.executor.memory=8g' jupyter/all-spark-notebook
-```
-
-Note that this is the same information expressed in a notebook in the Python case above. Once the kernel spec has your cluster information, you can test your cluster in an Apache Toree notebook like so:
-
-```scala
-// should print the value of --master in the kernel spec
-println(sc.master)
-
-// do something to prove it works
-val rdd = sc.parallelize(0 to 99999999)
-rdd.sum()
-```
-
```

Comment:
Maybe this content in `docs/using/specifics.md` can be moved to `examples/mesos.md`?

Reply:
That's an option. Another is to move them out entirely. One reason for deprecating these cluster-specific notes is that we don't have a way to maintain them.

Reply:
Ack  No problem with deleting.  Just seemed a waste to dispose of all the effort someone put into learning.  Anyway likely better suited to the Docker-recipes wiki page referenced in a comment to #626.